.circleci/run.sh

```diff
@@ -10,7 +10,6 @@ CIRCLE_STAGE=${CIRCLE_STAGE:-pic}
 CIRCLE_PROJECT_REPONAME=${CIRCLE_PROJECT_REPONAME:-dmd}
 BUILD="debug"
 DMD=dmd
-PIC=1
```

Comment:
What was the motivation to remove `PIC`? 
(Might also be useful to document this in  the commit message)

Reply:
Added this comment:

> PIC=1 was added during the NO_PIC/PIC transition before DMD defaulted to
PIC code on Linux. PIC=1 is has been the default for many years now and
on modern Linux distributions it's even the only way to execute
binaries.

Reply:
Thanks!